### PR TITLE
Persist flag become explicit argument to register environment

### PR DESCRIPTION
**Pull request by Ajay-MS**

### CODE REVIEW

**Comment by KrishnaAdityaB** [comment]
Looks good

File: Tasks/DeployAzureResourceGroup/task.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 64
+        "Patch": 65
```
Comment (kmkumaran) [inline Tasks/DeployAzureResourceGroup/task.json]: Should minimum agent version be updated?

**Comment by kmkumaran** [comment]
Looks fine.

**Comment by nigurr** [comment]
Looks fine

**Pull request merged**